docs/index.rst

```diff
@@ -1,51 +1,211 @@
+.. rubric:: Table of contents
 
-This client library implements the `Python Database API Specification v2.0`_
-(PEP 249), which defines a common interface for accessing databases in Python.
+.. contents::
+    :local:
+    :depth: 1
 
-It also includes the :ref:`CrateDB dialect <using-sqlalchemy>` for `SQLAlchemy`_.
 
-.. NOTE::
+************
+Introduction
+************
 
-    This is a basic CrateDB driver reference.
+The Python client library for `CrateDB`_ implements the Python Database API
+Specification v2.0 (`PEP 249`_), and also includes the :ref:`CrateDB dialect
+<using-sqlalchemy>` for `SQLAlchemy`_.
 
-    Check out the `sample application`_ (and the corresponding `sample
-    application documentation`_) for a practical demonstration of this driver
-    in use.
+The Python driver can be used to connect to both `CrateDB OSS`_ and `CrateDB
+Cloud`_, and is verified to work on Linux, macOS, and Windows. It is used by
+the `Crash CLI`_, as well as other libraries and applications connecting to
+CrateDB from the Python ecosystem. It is verified to work with CPython, but
+it has also been tested successfully with `PyPy`_.
 
-    For general help using the Python Database API or SQLAlchemy, please consult
-    `PEP 249`_, the `SQLAlchemy tutorial`_, or the `SQLAlchemy documentation`_.
+Please make sure to also visit the section about :ref:`other-options`, using
+the :ref:`crate-reference:interface-postgresql` interface of `CrateDB`_.
 
-.. SEEALSO::
 
-   The CrateDB Python client library is an open source project and is `hosted
-   on GitHub`_.
+*************
+Documentation
+*************
 
-.. rubric:: Table of contents
+For general help about the Python Database API, or SQLAlchemy, please consult
+`PEP 249`_, the `SQLAlchemy tutorial`_, and the general `SQLAlchemy
+documentation`_.
+
+For more detailed information about how to install the client driver, how to
+connect to a CrateDB cluster, and how to run queries, consult the resources
+referenced below. A dedicated section demonstrates how to use the :ref:`blob
+storage capabilities <crate-reference:blob_support>` of CrateDB.
 
 .. toctree::
-   :maxdepth: 2
+    :titlesonly:
+
+    getting-started
+    connect
+    query
+    blobs
+
+
+DB API
+======
+
+Install package from PyPI.
+
+.. code-block:: shell
+
+    $ pip install crate
+
+Connect to CrateDB instance running on ``localhost``.
+
+.. code-block:: python
+
+    >>> # Connect using DB API.
+    >>> from crate import client
+    >>> with client.connect("localhost:4200", username="crate") as connection:
+    >>>     cursor = connection.cursor()
+    >>>     cursor.execute("SELECT * FROM sys.summits;")
+    >>>     print(cursor.fetchall())
+    >>>     cursor.close()
+
+Connect to `CrateDB Cloud`_.
+
+.. code-block:: python
+
+    >>> # Connect using DB API.
+    >>> from crate import client
+    >>> client.connect("https://admin:iM_n1%0xRzShsapU4U(lDGOj@example.aks1.westeurope.azure.cratedb.net:4200")
```

Comment:
Might be nicer to read by using the `username`/`password` parameters:
```python
client.connect("https://example.aks1.westeurope.azure.cratedb.net:4200", username="admin", password="iM_n1%0xRzShsapU4U(lDGOj")
```

Reply:
Thanks. Fixed with 8235a75be0.